### PR TITLE
Added test for Fetcher fetchRoute + other stuff :)

### PR DESCRIPTION
**Pull request by p1pchenk0**
+ test for fetcher
* failing unit tests after updates

### CODE REVIEW

**Comment by p1pchenk0** [comment]
> Fab, thanks, good to merge or do you want to keep working on this branch tomorrow?

yes, I think it would be the best option to continue working here

File: src/runtime/api/auth.ts [hunk 1]
```diff
@@ -149,6 +149,10 @@ export default class Auth {
     return this.user.roles
   }
 
+  public get signedIn () {
```
Comment (silverbackdan) [inline src/runtime/api/auth.ts:152]: Brill, if you could also test this function that'd be great

Comment (p1pchenk0) [reply src/runtime/api/auth.ts:152]: tests added

Comment (p1pchenk0) [reply src/runtime/api/auth.ts:152]: added tests

File: src/runtime/composables/process.ts
[the remark this answers is not in the document]
Comment (p1pchenk0) [reply src/runtime/composables/process.ts:1]: added tests with one test marked as `todo`

File: src/runtime/composables/process.spec.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    process.client = true
+    process.server = false
+
+    expect(useProcess()).toEqual({
+      isClient: true,
+      isServer: false
+    })
+  })
+
+  test.todo('should return correct values for client/server flags IF env is server-side', () => {
```
Comment (silverbackdan) [inline src/runtime/composables/process.spec.ts:15]: Thank you, please add a comment "Disabled due to nuxt-vitest bug - track issue and progress here https://github.com/danielroe/nuxt-vitest/issues/162"

**Pull request merged**